plugins/outputs/influxdb/influxdb.go

```diff
@@ -48,6 +48,7 @@ type InfluxDB struct {
 	SkipDatabaseCreation bool              `toml:"skip_database_creation"`
 	InfluxUintSupport    bool              `toml:"influx_uint_support"`
 	tls.ClientConfig
+	TimePrecision string `toml:"time_precision"`
 
 	Precision string // precision deprecated in 1.0; value is ignored
```

Comment:
I'm not sure the reason, but I believe this functionality was deprecated intentionally.